portfolio/src/main/webapp/script.js

```diff
@@ -77,68 +82,78 @@ function addQuoteToDom(quote) {
+  ],
+  howard: [
+    38.9227,
+    -77.0194,
+    'My favorite HBCU, Howard University! This school has opened so many doors for me and allowed me to meet some of the funniest people in my life.',
+  ],
+
+  summerProgram: [
+    37.4104,
+    -122.0598,
+    'My first summer experience out on my own, The UNCF CS Summer Academy at Carnegie Mellon SV ',
+  ],
+
+  futureLocation: [
+    40.7484405,
+    -73.9878531,
+    'I have always wanted to live in NYC and hopefully after I graduate I will be able to move here.',
+  ],
+};
 
 function createMap() {
   map = new google.maps.Map(document.getElementById('map'), {
     center: { lat: 42.8864, lng: -78.8784 },
     zoom: 20,
     mapTypeId: 'satellite',
   });
+}
 
-  const buffalo = new google.maps.Marker({
-    position: { lat: 42.8864, lng: -78.8784 },
-    map: map,
-    title: 'Where I was born! Buffalo,NY',
-  });
-
-  const buffaloWindow = new google.maps.InfoWindow({
-    content:
-      'This is where I was born and raised. It is a pretty nice city with great scenery but I have to say there is not a lot fun places here other than the mall, Dave and Busters and the park.',
-  });
-  buffaloWindow.open(map, buffalo);
-
-  const howard = new google.maps.Marker({
-    position: { lat: 38.9227, lng: -77.0194 },
-    map: map,
-    title: 'Where I go to school! Howard University',
-  });
-
-  const howardWindow = new google.maps.InfoWindow({
-    content:
-      'My favorite HBCU, Howard University! This school has opened so many doors for me and allowed me to meet some of the funniest people of in life.',
-  });
-  howardWindow.open(map, howard);
+function createMarkers() {
+  for (const key in markerLocations) {
+    if (key) {
+      const marker = new google.maps.Marker({
+        position: {
+          lat: markerLocations[key][0],
+          lng: markerLocations[key][1],
+        },
+        map: map,
+      });
+      createInfoWindows(marker, key);
+    }
+  }
+}
 
-  const summerProgram = new google.maps.Marker({
-    position: { lat: 37.4104, lng: -122.0598 },
-    map: map,
-    title:
-      'My first summer experience out on my own, The UNCF CS Summer Academy at Carnegie Mellon SV ',
+function createInfoWindows(marker, key) {
+  const infoWindow = new google.maps.InfoWindow({
+    content: markerLocations[key][2],
   });
+  infoWindow.open(map, marker);
+}
 
-  const summerProgramWindow = new google.maps.InfoWindow({
-    content:
-      'My first time in Californa! This experience is so near and dear to my heart and it lead me to me meet so many friends from different HBCUs',
-  });
+function changeLocation(updatedLat, updatedLong) {
+  map.setCenter(new google.maps.LatLng(updatedLat, updatedLong));
+}
 
-  summerProgramWindow.open(map, summerProgram);
+document.getElementById('cali').addEventListener('click', function () {
```

Comment:
Not a needed change but for future reference, these can also be iterated through like `createMarkers` in a similar function like `installEventListeners`.

Reply:
Noted!